apps/design-system/registry/examples.ts

```diff
@@ -1039,21 +1069,6 @@ export const examples: Registry = [
     type: 'components:example',
     files: ['example/text-confirm-dialog-demo.tsx'],
   },
-  {
-    name: 'text-confirm-dialog-with-info-alert',
-    type: 'components:example',
-    files: ['example/text-confirm-dialog-with-info-alert.tsx'],
-  },
-  {
-    name: 'text-confirm-dialog-with-warning-alert',
-    type: 'components:example',
-    files: ['example/text-confirm-dialog-with-warning-alert.tsx'],
-  },
-  {
-    name: 'text-confirm-dialog-with-destructive-alert',
-    type: 'components:example',
-    files: ['example/text-confirm-dialog-with-destructive-alert.tsx'],
-  },
```

Comment:
I removed these because the Text Confirm Dialog is only ever really relevant when you’re making a destructive action. Hence why the user needs to painstakingly confirm a string.